static/app/views/explore/components/addToDashboardButton.tsx

```diff
@@ -87,7 +87,7 @@ export function AddToDashboardButton({visualizeIndex}: AddToDashboardButtonProps
     <Tooltip title={t('Add to Dashboard')}>
       <Button
         size="sm"
-        icon={<IconDashboard />}
+        icon={<IconEllipsis />}
```

Comment:
@doralchan is the intention to make the ellipsis open a context menu with an option to add to dashboard? If so, this PR is incorrect as the ellipsis will directly add to dashboard.

Reply:
@narsaynorath made the same point but I decided to just wait for him to make the changes